features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Details/Endpoints/AWSLambda/Credentials.jsx

```diff
@@ -190,6 +279,37 @@ export default function Credentials(props) {
                         endpointsDispatcher({ action: 'set_awsCredentials', value: newEndpointConfig });
                     }}
                 />
+                <FormControl
+                    required
+                    margin='normal'
+                    variant='outlined'
+                    disabled={endpointConfig.access_method === 'role-supplied'}
+                >
+                    <InputLabel ref={inputLabel} id='demo-simple-select-outlined-label'>
```

Comment:
Shall we remove this id if not in use, or change it to a proper name `demo-simple-select-outlined-label`

Reply:
Sure, I think that id can be removed. Thanks for spotting it!